src/components/Person/Person.jsx

```diff
@@ -1 +1,37 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = (obj) => {
+  const objKeys = { ...obj };
+  const people = objKeys.person;
+
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = people;
+
+  const partner = sex === 'm'
+    ? 'wife'
+    : 'husband';
+
+  const withPartner = `${partnerName} is my ${partner}`;
+  const checkForPartner = isMarried === false
+    ? 'I am not married'
+    : withPartner;
+
+  const withAge = age
+    ? <p className="Person__age">{`I am ${age}`}</p>
+    : '';
+
```

Comment:
You can leave only partner variable and make all the other checks on return.
I think it will make your code cleaner, but it's up to you